### PR TITLE
Fix group chat info labels

### LINKED ISSUE #8105: No labels for icons are shown on Group Info screen
### Description
_Type_: Bug
_Summary_: no icons are displayed on "Group Info" screen for group chats

#### Expected behavior
Can see labels for options

#### Actual behavior
Screenshot!UNITO-UNDERSCORE!2019-05-02-10-14-45

### Reproduction

* Open Status
* Create account
* Add contact
* Create group chat
* Open three dots > Group info

### Additional Information
[comment]: # (Please do your best to fill this out.)

* Status version: nightly 02/05/2019
* Operating System: Android, IOS

### PR DESCRIPTION
**Pull request by vitvly**
Fixes #8105

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/profile/group_chat/styles.cljs [hunk 1]
```diff
@@ -9,7 +9,7 @@
   {:background-color (colors/alpha colors/blue 0.1)
    :border-radius    50})
 
-(def action-label
+(defn action-label [with-subtext?]
```
Comment (jeluard) [inline src/status_im/ui/screens/profile/group_chat/styles.cljs]: Argument not used?

**Review by jeluard — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/profile/group_chat/styles.cljs (hunk 1 above)
Comment (vitvly) [reply src/status_im/ui/screens/profile/group_chat/styles.cljs]: Yes, it's just that list action label styles are fns now. Rename arg to `_`?

Comment (jeluard) [reply src/status_im/ui/screens/profile/group_chat/styles.cljs]: Ah ok! Yes please! And maybe a comment so that no-one removes it by accident?

**Review by jeluard — APPROVED** [review]

File: src/status_im/ui/screens/profile/group_chat/styles.cljs (hunk 1 above)
Comment (vitvly) [reply src/status_im/ui/screens/profile/group_chat/styles.cljs]: Renamed and added a comment.

**Comment by asemiankevich** [comment]
@siphiuel could you please rebase? iOS build is expired and now i can not build (pedro did some changes), rebase should help :)

**Comment by vitvly** [comment]
@asemiankevich sure, rebased and pushed.

**Pull request merged**